README.md

```diff
@@ -20,6 +20,15 @@ You can use the framework in various services such as social networks, call cent
 * The framework is being actively maintained and thoroughly tested. The team is open to suggestions and quickly reacts to bug reports.
 
 # Quick Start
+
+## System Requirements
+
+- Supported operating systems include Ubuntu 18.04+, Windows 10+ (partial support), and MacOS Big Sur (partial support);
```

Comment:
Why partial support?
We don't test them because of docker. I don't think there's anything else we don't support (and docker can be launched on these platforms).